samcli/commands/package/package_context.py

```diff
@@ -47,6 +48,8 @@ class PackageContext:
         "\n"
     )
 
+    uploaders: Uploaders
```

Comment:
I have no intention to make it class/static variable. It can also be instance variable. See mypy doc:

<img width="765" alt="image" src="https://user-images.githubusercontent.com/3804518/104962473-579b7a00-598d-11eb-813c-5f4be027836f.png">

https://mypy.readthedocs.io/en/stable/class_basics.html

Reply:
I see, thanks!